machine/tool/api/src/main/java/org/qbicc/machine/tool/CCompilerInvoker.java

```diff
@@ -120,6 +120,7 @@ default void addDefinedSymbol(String name) {
     enum SourceLanguage {
         C,
         ASM,
+        IR,
```

Comment:
Minor detail, but this should be `LLVM_IR` since LLVM is not assumed at the C compiler level.

Reply:
done!